models/logs.js

```diff
@@ -35,10 +35,22 @@ const fetchLogs = async (query, param) => {
     let call = logsModel.where("type", "==", param);
     Object.keys(query).forEach((key) => {
       // eslint-disable-next-line security/detect-object-injection
```

Comment:
eslint-disable is still required, as you are changing the below condition.
check once after removing this condition.

Reply:
Need some more clarity on this @alishvagadiya